src/CommunityToolkit/Xamarin.CommunityToolkit/Views/DrawingView/DrawingView.shared.cs

```diff
@@ -0,0 +1,93 @@
+﻿using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.IO;
+using System.Linq;
+using System.Windows.Input;
+using Xamarin.Forms;
+
+namespace Xamarin.CommunityToolkit.UI.Views
+{
+	public class DrawingView : View
+	{
+		const int minValueGranularity = 5;
+
+		public static readonly BindableProperty GranularityProperty =
+			BindableProperty.Create(nameof(Granularity), typeof(int), typeof(DrawingView), minValueGranularity, coerceValue: CoerceValue);
+
+		public static readonly BindableProperty EnableSmoothedPathProperty =
+			BindableProperty.Create(nameof(EnableSmoothedPath), typeof(bool), typeof(DrawingView), default(bool));
+
+		public static readonly BindableProperty LineColorProperty =
+			BindableProperty.Create(nameof(LineColor), typeof(Color), typeof(DrawingView), Color.Default);
+
+		public static readonly BindableProperty LineWidthProperty =
+			BindableProperty.Create(nameof(LineWidth), typeof(float), typeof(DrawingView), 5f);
+
+		public static readonly BindableProperty ClearOnFinishProperty =
+			BindableProperty.Create(nameof(ClearOnFinish), typeof(bool), typeof(DrawingView), default(bool));
+
+		public static readonly BindableProperty PointsProperty = BindableProperty.Create(
+			nameof(Points), typeof(ObservableCollection<Point>), typeof(DrawingView), new ObservableCollection<Point>(), BindingMode.TwoWay);
+
+		public static readonly BindableProperty DrawingCompletedCommandProperty = BindableProperty.Create(
+			nameof(DrawingCompletedCommand), typeof(ICommand), typeof(DrawingView), default(ICommand));
+
+		public ObservableCollection<Point> Points
```

Comment:
Should we make this a `ObservableCollection`? Shouldn't we take something more low-level and make it a choice for people to use this?